simplified-app-shared/build.gradle

```diff
@@ -15,33 +15,48 @@ android {
   }
 }
 
+repositories {
```

Comment:
What is jcenter?

Reply:
Some of the stuff I was pulling in comes from there.  See this:  https://stackoverflow.com/questions/45818613/trove4j-library-cannot-be-resolved .  jcenter used to get pulled in by some of the child libraries.  Not that "implementation" doesn't do recursive dependencies, you have to add it everywhere.